### ISSUE TITLE
Making cat bat?

### ISSUE THREAD

**Issue by Haveyougonemad** [sharkdp/bat#1196]
Is there anyway to make bat into cat? like making bat command into cat bc im used to do cat rather bat so... alias weren't the best way of doing it..

**Comment by kurucu** [comment]
Could you not create an alias?

**Comment by Haveyougonemad** [comment]
> 
> 
> What was wrong with an alias?

i use zsh and bash, zsh isn't default shell by chsh, i modified .bashrc to execute it when lauching then when i setup alise for both bash and zsh they conflit, and single doesn't work aswell, and can't get arguements.

**Comment by sharkdp** [comment]
Sorry, but it's hard to follow what you really want and what you tried so far. Setting
```
alias cat=bat
```
in both `.bashrc` and `.zshrc` works perfectly well. Also, this does not seem to be an issue with `bat` itself. If it's really a problem with your shells, you should consider asking somewhere else.

**Comment by Haveyougonemad** [comment]
> 
> 
> Sorry, but it's hard to follow what you really want and what you tried so far. Setting
> 
> ```
> alias cat=bat
> ```
> 
> in both `.bashrc` and `.zshrc` works perfectly well. Alias do not "conflict" between different shells. Also, this does not seem to be an issue with `bat` itself. If it's really a problem with your shells, you should consider asking somewhere else.

Hmm idk, when i typed cat it says unknown command then, it is working now TYSM :) have a nice day.
